src/olympia/reviewers/management/commands/review_reports_sql/addon/breakdown.sql

```diff
@@ -2,30 +2,50 @@ SELECT 'All Reviewers' AS `Group`,
        IFNULL(FORMAT(SUM(aa.weight), 0), '-') AS `Total Risk`,
        IFNULL(FORMAT(AVG(aa.weight), 2), '-') AS `Average Risk`,
        FORMAT(COUNT(*), 0) AS `Add-ons Reviewed`
-FROM `reviewer_scores` `rs`
```

Comment:
I suppose we can delete the separate stats for volunteers as well (=this entire file).

It would be great if the total for all reviewers recorded here could be shown in the "Weekly Add-on Reviews" section instead. To make it easier to understand, we can drop the "5 Reviews or more" constraint.

The same applies to content reviews.

Reply:
@wagnerand to confirm, we'd just end up with 3 sql files:
- addon/weekly.sql (without the >=5 constraint)
- addon/risk.sql 
- content/weekly.sql (without the >=10 constraint)

Reply:
Yes. Can we the weekly section include the totals?

Reply:
https://github.com/mozilla/addons-server/pull/20060/commits/9ac2259f4f75ff81533c2cd193acfb1e6d9fead1